terraform_utils/json.go

```diff
@@ -15,8 +16,18 @@ func jsonPrint(data interface{}) ([]byte, error) {
 	}
 	// We don't need to escape > or <
 	s := strings.Replace(string(dataJsonBytes), "\\u003c", "<", -1)
-	s = strings.Replace(s, "\\<", "<", -1) // fix broken double escaping
+	s = regexp.MustCompile(`.?\\<`).ReplaceAllStringFunc(s, escapingBackslashReplacer("<"))
```

Comment:
MustCompile this Compile with panic for init on start application.
Compile has very bad performance.

Best practices use MustCompile as global variable. 
Load once and return panic on load if you have bad regex.

In this case we dont need run Compile twice on each resource and we have sometimes many resources(10000+). 

Better to move mustcompile to global variable

Reply:
Thanks, fixed